### ISSUE TITLE
[Suggestion] Read and Import Vanilla (AE) Saves

### ISSUE THREAD

**Issue by JoLiKMC** [FTL-Hyperspace/FTL-Hyperspace#201]
Greetings.

First and foremost, I really like this project.  The added functionality of this mod, such as "speed up" and the ability to add even more ships in a non-intrusive way, is really great!  However, I do have a question / suggestion.

Is is currently / would it be possible to **import save games from the base _Advanced Edition_ game?**  I recently spent a lot of time unlocking a bunch of ships and ship layouts in the normal game with the hopes of using those saves in _Hyperspace_ (for mods and things), but the data format between `ae_prof.sav` and `hs_prof.sav` has significant changes which make it seemingly impossible to go from one to the other.

I know I can use _Hyperspace_ console commands to unlock ships and achievements, but I'd prefer not to do that because, for one, I don't know the correct data names for each ship and its layouts – and also don't know how to crack into `ftl.dat` to find out – and for another, I already have a perfectly functional save in normal ol' _Advanced Edition_, so… yeah.

Thank you for all your hard work!
Hoping to hear good things back~!

**Comment by JoLiKMC** [comment]
I am an absolute _dingbat_.

Okay, so.  I tried to run _Hyperspace_ using a different data directory.  In doing so, it obviously didn't notice my `ae_prof.sav` file because it wasn't looking in the usual place.  Now, I was messing around with some console commands and once I was done, I erased my `hs_prof.sav` file.  Upon reloading the game, it crashed at the title screen.  After erasing my `hs_version.sav` file, too, the game was able to load normally… except, for some reason, I had access to every ship I'd unlocked on my _Advanced Edition_ save.  Why?  Well… I'd copied that save to the data directory I was using for testing stuff.  And, when _Hyperspace_ didn't see any of its own save files but _did_ see the _Advanced Edition_ save… **it imported and converted that data.**

So, to recap…
- I ran _Hyperspace_ using a different data directory.  Because of this, it didn't see an _Advanced Edition_ save and generated fresh files.
- I copied my `ae_prof.sav` file over to that data directory.  No changes.
- I erased my _Hyperspace_ saves entirely, but kept my _Advanced Edition_ save, then loaded the game, again.
- Surprise surprise, _Hyperspace_ loaded and converted the _Advanced Edition_ data.  No fuss, no muss.

As I said… complete dingbat, this one.  (,:
